### PR TITLE
Migrating the remaining Prefect flows

### PR DESCRIPTION
**Pull request by Charlie-Henry**
Migrates the last remaining Prefect flows over to Airflow
- [Traffic incidents](https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-traffic-incident-reports/traffic-incident-reports.py)
- [ROW currently active permits](https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-executive-dashboard/row_active_permits.py)
- [ROW permit weekly totals](https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-executive-dashboard/row_data_publishing.py)
- [Parking data processing](https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-parking-data/parking_data_processing.py) I ended up not using the dynamic args that we had in Prefect. 
- [Bond reporting](https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-bond-reporting/atd-bond-reporting.py)

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13192
https://github.com/cityofaustin/atd-data-tech/issues/12860

## Associated repos
[atd-parking-data](https://github.com/cityofaustin/atd-parking-data)
[atd-traffic-incident-reports](https://github.com/cityofaustin/atd-traffic-incident-reports)
[atd-bond-reporting](https://github.com/cityofaustin/atd-bond-reporting)
[atd-executive-dashboard](https://github.com/cityofaustin/atd-executive-dashboard)

## Testing

**Steps to test:**
1. Checkout this branch
2. docker compose up
3. Give these ETLs a shot! Note that parking ETL can take a while 🐌 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_traffic_incident_reports.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=5),
```
Comment (chiaberry) [inline dags/atd_traffic_incident_reports.py]: the dag runs every 5 mins, should the timeout be shorter?

**Comment by johnclary** [comment]
i opened https://github.com/cityofaustin/atd-bond-reporting/pull/5 because i can't test the bond data DAG locally.

**Comment by johnclary** [comment]
Also: https://github.com/cityofaustin/atd-executive-dashboard/pull/5

File: dags/atd_traffic_incident_reports.py (hunk 1 above)
Comment (Charlie-Henry) [reply dags/atd_traffic_incident_reports.py]: yes, dropped to 1 minute now 👍

**Review by chiaberry — APPROVED** [review]
Thanks, you are [a real hero](https://soundcloud.com/electricyouthmusic/college-and-electric-youth-a-real-hero).

I opened a PR to also update the atd-traffic-incident-reports docker image github workflow

**Review by johnclary — APPROVED** [review]
hooray! thank you @Charlie-Henry!

🙌 🚢 👋

File: dags/atd_bond_reporting.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        mount_tmp_dir=False,
+        retries=3,
+        retry_delay=duration(seconds=60),
+    )
+
+    t6 = DockerOperator(
+        task_id="bond_data_processing",
+        image=docker_image,
+        auto_remove=True,
+        command="python atd-bond-reporting/bond_calculations.py",
```
Comment (mddilley) [inline dags/atd_bond_reporting.py:177]: Caught one warning on this one but I am not versed in 🐼 
```
INFO - atd-bond-reporting/bond_calculations.py:300: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead
See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy```

**Review by mddilley — COMMENTED** [review]
Thanks for moving all of these over! I tested each one that was multiarch, and I saw all greens in Airflow. 🙌 🚀 🎉 🙏

**Pull request merged**